frontend/src/pages/Index.vue

```diff
@@ -1,13 +1,15 @@
 <template>
   <q-page class="flex flex-center">
```

Comment:
```suggestion
  <q-page class="flex q-mt-sm"">
```

Essentially, each of the Page templates would have the same `q-page` wrapper:

```html
<template>
  <q-page class="flex q-mt-sm">
    <front-page />
  </q-page>
</template>
```

Is there a way to share the `<q-page class="flex q-mt-sm">`across pages so they all behave the same without redundant code?

Reply:
Done